upgrade_test.py

```diff
@@ -236,14 +242,11 @@ def upgrade_node(self, node, upgrade_sstables=True):
                     node.remoter.run(
                         r'sudo apt-get dist-upgrade {} -y '
                         r'-o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" '.format(scylla_pkg))
-        if self.params.get('test_sst3'):
-            node.remoter.run("echo 'enable_sstables_mc_format: true' |sudo tee --append /etc/scylla/scylla.yaml")
-        if self.params.get('test_upgrade_from_installed_3_1_0'):
-            node.remoter.run("echo 'enable_3_1_0_compatibility_mode: true' |sudo tee --append /etc/scylla/scylla.yaml")
         check_reload_systemd_config(node)
         # Current default 300s aren't enough for upgrade test of Debian 9.
         # Related issue: https://github.com/scylladb/scylla-cluster-tests/issues/1726
         node.run_scylla_sysconfig_setup()
+        self._update_scylla_yaml_on_node(node_to_update=node, updates=scylla_yaml_updates)
```

Comment:
why here and not where `test_sst3` was before ?

otherwise the call there for `check_reload_systemd_config` seems kind of pointless

Reply:
I think this only matters if we touch the unit definitions for `systemd`. `scylla.yaml` is read when the service is restarted, so I put the update method before the restart.

Reply:
Since those flags weren't used for a while, I guess it doesn't make much difference.

But we'll see once all the upgrade tests would run